packages/docs/src/data/team.json

```diff
@@ -112,5 +112,65 @@
     "location": "USA",
     "name": "Elijah Kotyluk",
     "team": "core"
+  },
+  "yooneskh": {
+    "discord": "YoonesKh#7826",
+    "focus": [
+      "[vuetifyjs](https://github.com/vuetifyjs)",
+      "[vuetifyjs/Community Suppport](https://discord.com/invite/vuetify-340160225338195969)"
+    ],
+    "languages": [
+      "English",
+      "Persian"
+    ],
+    "twitter": "yooneskh",
+    "linkedin": "yooneskh ",
+    "location": "Iran",
+    "name": "Yoones Khoshghadam",
+    "team": "core"
+  },
+  "": {
```

Comment:
should this be empty?

Reply:
she doesn't have a github account, so I have no choice.